streamly.cabal

```diff
@@ -709,3 +707,8 @@ library
     -- tests fail
     if flag(dev) && flag(inspection)
       build-depends: inspection-and-dev-flags-cannot-be-used-together
+      
+    if flag(dev)
+      exposed-modules:  Streamly.Internal.Data.SmallArray
+                      , Streamly.Internal.Data.SmallArray.Type
+
```

Comment:
Would be better if we put this before the build-depends section, next to other modules.